deploy/roles/r-studio-server/tasks/main.yml

```diff
@@ -26,6 +26,7 @@
     state: "{{ rstudio_server_package_state }}"
     disable_gpg_check: yes
     validate_certs: no
+  when: upgrade_rstudio
```

Comment:
@stirlingv I think we already have an `update_rstudio` variable, do we need a separate one?

Reply:
I think that variable is for something different. I left `update_rstudio=false` when running the upgrade. If you don't think it matters, I'll happily modify to all use the same variable.

Reply:
Ah, technically `update_rstudio` should be `True` when upgrading.  This ensures that all users are disconnected before the upgrade happens, which prevents some issues from occurring.

Reply:
oh, I get it! sorry about that. I'll make the changes.